lib/pronto/formatter/text_formatter.rb

```diff
@@ -14,9 +30,30 @@ def location(message)
         line = message.line
         lineno = line.new_lineno if line
         path = message.path
-        commit_sha = message.commit_sha[0..6] if message.commit_sha
+        commit_sha = message.commit_sha
+
+        if path || lineno
```

Comment:
This was something I was wondering about and decided to go with some short and specialized methods that also make rubocop & reek happier. 

Indeed you are correct that they also introduce layers/levels of abstractions and increase the cognitive load when comprehending such a simple code. I've addressed this in my latest commit.